### PR TITLE
feat: `translate` module coverage

### PR DESCRIPTION
**Pull request by FBruzzesi**
## What type of PR is this? (check all applicable)

- [ ] 💾 Refactor
- [ ] ✨ Feature
- [ ] 🐛 Bug Fix
- [ ] 🔧 Optimization
- [ ] 📝 Documentation
- [x] ✅ Test
- [ ] 🐳 Other

## Checklist

- [x] Code follows style guide (ruff)
- [x] ~~Tests added~~ These are the tests
- [ ] Documented the changes

### CODE REVIEW

File: tests/translate/from_native_test.py [hunk 1]
```diff
@@ -0,0 +1,147 @@
...
+all_frames = [*eager_frames, *lazy_frames]
+
+all_series = [
+    series_pd,
+    series_pl,
+    series_mpd,
+    series_pa,
+    MockSeries(),
+]
+
```
Comment (FBruzzesi) [inline tests/translate/from_native_test.py:67]: I was not able to use conftest to do what I wanted 🙈

**Comment by MarcoGorelli** [comment]
aww thank you so much, really appreciate this

**Comment by FBruzzesi** [comment]
@MarcoGorelli should I consider tackle #470 as well in here? 

Invalid combinations should be just `series_only=True and allow_series = False`?

**Review by MarcoGorelli — APPROVED** [review]
🙏 🙌 💯

**Pull request merged**